staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -760,6 +762,13 @@ type PersistentVolumeClaimVolumeSource struct {
 	Hotpluggable bool `json:"hotpluggable,omitempty"`
 }
 
+type MemoryDumpVolumeSource struct {
+	// PersistentVolumeClaimVolumeSource represents a reference to a PersistentVolumeClaim in the same namespace.
+	// Directly attached to the virt launcher
+	// +optional
+	PersistentVolumeClaim *PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty"`
```

Comment:
I wonder if this struct would be better since I doubt there will be any targets to memory dump other than pvc?
```golang
type MemoryDumpVolumeSource struct {
    PersistentVolumeClaimVolumeSource `json:",inline"`
}
```

Reply:
yeah might be a good suggestion